### PR TITLE
Create calculator layout

### PR DESCRIPTION
**Pull request by yasinabdmahmood**
### Changes implemented on the projec

- Deleted all the boilerplate from CRA (text, images, styles).
- Created a directory called `components`.
- Inside `components`, created a new `Calculator.js` file.
- In `Calculator.js`, created a React component that matches the required design :

- Developed the UI of the calculator.
- Used class-based components (All components are class-based).
- imported it and use it in your main component (App.js).
- Tested it in the browser; it renders without problems.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```
Comment (victorokoroji) [inline src/App.js]: - [ ] Kindly ensure that your `<App />` component is written as a class-based component according to the project requirement that all component must be class-based 👍🏽

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Item from './Item';
+// eslint-disable-next-line react/prefer-stateless-function
+class Calculator extends React.Component {
```
Comment (victorokoroji) [inline src/components/Calculator.js:4]: - [ ] I think you should make your calculator design look like the design specified for this project by removing the thick black borders from the buttons present in your design 😄 

Your Design 👇🏽 

image


Project Design 👇🏽 

image

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @yasinabdmahmood, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅  
- Linters are working properly ✅ 


## Required Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShahierNashaat — APPROVED** [review]
Hi @yasinabdmahmood, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Good overall design ✔️
- Your code is clean and well documented ✔️

# Approved :ok_hand: :ok_hand: :ok_hand:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**